india_compliance/public/scss/_return_tool.scss

```
@@ -0,0 +1,66 @@
+    padding-right: 0;
+}
+
+// the tabbed data area, and the toolbar frappe draws beside the tab titles
+.gst-return-tabs {
+    .section-body {
+        margin-top: 0;
+    }
+
+    .form-tabs-list {
+        @include full-bleed-divider;
+        display: flex;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)